package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-runner",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
Could we bump the version in a dedicated PR? I'd like to get in a really small fix for start-up module loading which doesn't really deserve its own version :P